### PR TITLE
fix(cli): windows EOL

### LINKED ISSUE #2551: which unix2dos 'which' is not recognized error
### Describe the bug

From console:

npx ignite-cli@latest new YoldaClient

✅ What bundle identifier? · com.yolda_client
✅ Where do you want to start your project? · C:\CODING\LEARNING\MOBILE APPS\yolda-client
✅ Do you want to initialize a git repository? (Y/n) · Yes
✅ Remove demo code? We recommend leaving it in if it's your first time using Ignite (y/N) · No
✅ Which package manager do you want to use? · pnpm
✅ Do you want to install dependencies? (Y/n) · Yes

    █ Creating YoldaClient using Ignite 8.11.1
    █ Powered by  ∞ Infinite Red  (https://infinite.red)
    █ Package Manager: pnpm
    █ Bundle identifier: com.yolda_client
    █ Path: C:\CODING\LEARNING\MOBILE APPS\yolda-client
    ────────────────────────────────────────────────────

    🖨  3D-printing a new React Native app
    🎨 Getting those last few details perfect
    🧶 Installing pnpm dependencies (wow these are heavy)

The following error occurred:

    Error: Command failed with exit code 1: which unix2dos
'which' is not recognized as an internal or external command,
operable program or batch file.

### Ignite version

8.11.1

### Additional info

```
System
  platform           win32
  arch               x64
  cpu                12 cores       AMD Ryzen 5 5600X 6-Core Processor
  directory          yolda-client   C:\CODING\LEARNING\MOBILE APPS\yolda-client

JavaScript (and globally-installed packages)
  node               19.0.0                                                                              C:\Program Files\nodejs\node.EXE
  npm                9.6.2                                                                               C:\Program Files\nodejs\npm.CMD
    eas-cli          5.6.0

    nodemon          2.0.22

    npm              9.6.2

    pnpm             8.9.2

    tile-extruder    2.1.0

    typescript       5.1.3

  yarn               -                                                                                   not installed

  pnpm               8.9.2                                                                               C:\Users\dur2d\AppData\Local\pnpm\pnpm.EXE
                     <no pnpm global package info available>

  expo               48.0.21 -> .\node_modules\.pnpm\expo@48.0.21_@babel+core@7.23.3\node_modules\expo   managed


Ignite
  ignite-cli         8.11.1       C:\Users\dur2d\AppData\Local\npm-cache\_npx\e31027f3785124a8\node_modules\.bin\ignite.CMD
  ignite src         build        C:\Users\dur2d\AppData\Local\npm-cache\_npx\e31027f3785124a8\node_modules\ignite-cli\build

Android
  java               21.0.1       C:\Program Files\Common Files\Oracle\Java\javapath\java.EXE
  android home       -            C:\Users\dur2d\AppData\Local\Android\Sdk

Tools
  git                git version 2.42.0.windows.2   C:\Program Files\Git\cmd\git.EXE
```

### Comment
Also happens with: `npx ignite-cli@latest new PizzaApp --yes`:

    █ Creating PizzaApp using Ignite 8.11.1
    █ Powered by  ∞ Infinite Red  (https://infinite.red)
    █ Package Manager: npm
    █ Bundle identifier: com.pizzaapp
    █ Path: C:\CODING\LEARNING\MOBILE APPS\PizzaApp
    ────────────────────────────────────────────────────

    🖨  3D-printing a new React Native app
    🎨 Getting those last few details perfect
    🧶 Installing npm dependencies (wow these are heavy)

The following error occurred:

    Error: Command failed with exit code 1: which unix2dos
'which' is not recognized as an internal or external command,
operable program or batch file.

### PR DESCRIPTION
**Pull request by frankcalise**
## Please verify the following:

- [ ] `yarn test` **jest** tests pass with new tests, if relevant
- [ ] `README.md` has been updated with your changes, if relevant

## Describe your PR

- Closes #2551
- Not every dev machine on Windows has the `unix2dos` utility installed via `git` apparently, so we changed it up to just change line endings more manually
- Also added a try/catch around this region specifically, since it doesn't need to kill the entire `new` command

### CODE REVIEW

File: src/commands/new.ts [hunk 1]
```diff
@@ -566,35 +566,21 @@ export default {
...
-          // inspect the results of conversion and log
-          results.forEach((result) => {
-            if (result.status === "rejected") {
-              warnAboutEOL = true
-              log(`Error converting EOL: ${JSON.stringify(result.reason)}`)
-            }
+          log(`templates to change EOL: ${templates}`)
+          templates.map(async (file) => {
+            log(`Converting EOL for ${file}`)
+            let template = read(file)
```
Comment (jamonholmgren) [inline src/commands/new.ts:579]:
```suggestion
            const template = read(file)
```

Comment (jamonholmgren) [reply src/commands/new.ts:579]: Really minor nit, doesn't even need to be accepted, but a suggestion.

**Review by jamonholmgren — COMMENTED** [review]
Good to go!

File: src/commands/new.ts (hunk 1 above)
Comment (frankcalise) [reply src/commands/new.ts:579]: can I const that? reassigning it on the next line

**Pull request merged**